ENIGMAsystem/SHELL/Universal_System/highscore_functions.cpp

```diff
@@ -112,7 +112,7 @@ void highscore_set_border(bool show) {enigma::highscore_border=show;}
 
 void highscore_set_font(std::string name, int size, int style) {
     //not cross platform compatible?
-    show_error("Warning function: highscore_set_font doesn't work, please use draw_set_font", false);
+    DEBUG_MESSAGE("Warning function: highscore_set_font doesn't work, please use draw_set_font", MESSAGE_TYPE::M_ERROR);
```

Comment:
This might as well be a warning instead of saying "warning." Also, wgaf if the font is wrong as a consequence.